src/Containers.jsx

```diff
@@ -40,19 +42,84 @@ class Containers extends React.Component {
         }));
     }
 
-    //TODO
-    stopContainer(container) {
-        return undefined;
+    stopContainer(container, timeout) {
+        document.body.classList.add('busy-cursor');
+        if (!timeout) {
+            timeout = 10;
+        }
+        utils.varlinkCall(utils.PODMAN, "io.podman.StopContainer", {name: container.ID, timeout: timeout})
```

Comment:
Hint: You could write this more succinctly as `timeout: timeout || 10`, and drop the above `if`. (you don't have to though, matter of preference)